README.md

```diff
@@ -1,2 +1,43 @@
-# awesome-books-ES6
-awesome  books with modules
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books with Modules
+
+> add and remove books from list
+
+## Built With
+
+- Html
+- Css
+- Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://arturgouveia1970.github.io/awesome-books-ES6/)
+
+## Authors
+
+👤 **Artur Gouveia**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] **[REQUIRED]** Kindly fix the MIT license page because right now it shows a 404 error page.

Reply:
@julie-ify, how can I do that?  was there a lesson on this subject. I don't remenber it.

Reply:
sorry, @julie-ify, I have done it. I actually that it was necessary to do that before today. thank you.

Reply:
Hi @Arturgouveia1970, good job fixing the links doing this will make your readme more professional 👍🏿